src/org/exist/xquery/parser/XQueryTree.g

```diff
@@ -470,7 +470,7 @@ throws PermissionDeniedException, EXistException, XPathException
 			{
 				PathExpr enclosed= new PathExpr(context);
 				SequenceType type= null;
-				QName qn = QName.parse(staticContext, qname.getText());
+				QName qn = QName.parse(staticContext, qname.getText(), null);
```

Comment:
Instead of `null` probably better to show your intention and use `XMLConstants.NULL_NS_URI`